# This file is part of HiGFXback

# configure
sh ./configure.sh --prefix

# build
./build.sh

# install
install -d $DESTDIR/bin
install pkg-config $DESTDIR/bin
install -d $DESTDIR/share/aclocal
install -m 644 pkg.m4 $DESTDIR/share/aclocal

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/pkg-config-lite-build.pc << EOF
Name: pkg-config-lite
Version: 0.28-1
Description: Tool to retrieve information about installed packages

devel=\\
/bin/pkg-config \\
/share/aclocal/pkg.m4
EOF

exit
--- pkg-config-lite-0.28-1.orig/build.sh.in
+++ pkg-config-lite-0.28-1/build.sh.in
@@ -0,0 +1,34 @@
+#!/bin/sh
+# Shell script to build pkg-config-lite in the absence of any `make' program.
+# @configure_input@
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+
+VERSION='@VERSION@'
+
+srcdir='@srcdir@'
+CC='@CC@'
+CFLAGS='@CFLAGS@'
+
+prefix='@prefix@'
+exec_prefix=`eval echo @exec_prefix@`
+libdir=${exec_prefix}/lib
+
+ENABLE_INDIRECT_DEPS=`test @use_indirect_deps@ = yes && echo TRUE || echo FALSE`
+
+CFLAGS="$CFLAGS -DENABLE_INDIRECT_DEPS=$ENABLE_INDIRECT_DEPS -DPKG_CONFIG_PC_PATH=\"${libdir}/pkgconfig:${prefix}/share/pkgconfig\" -DPKG_CONFIG_SYSTEM_INCLUDE_PATH=\"${prefix}/include\" -DPKG_CONFIG_SYSTEM_LIBRARY_PATH=\"${libdir}:/lib\" -DVERSION=\"$VERSION\""
+
+echo $CC $CFLAGS $srcdir/glib-snippet.c $srcdir/pkg.c $srcdir/parse.c $srcdir/main.c -o pkg-config
+$CC $CFLAGS $srcdir/glib-snippet.c $srcdir/pkg.c $srcdir/parse.c $srcdir/main.c -o pkg-config
--- pkg-config-lite-0.28-1.orig/configure.sh
+++ pkg-config-lite-0.28-1/configure.sh
@@ -0,0 +1,71 @@
+#!/bin/sh
+# Shell script to configure pkg-config-lite without relying on `autotools'.
+
+# Copyright (C) 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2000, 2001, 2002,
+# 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010 Free Software Foundation, Inc.
+#
+# This file is part of GNU Make.
+#
+# GNU Make is free software; you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 3 of the License, or (at your option) any later
+# version.
+#
+# GNU Make is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.  If not, see <http://www.gnu.org/licenses/>.
+
+VERSION=0.28
+
+srcdir=.
+
+CC=${CC-gcc}
+CFLAGS=${CFLAGS:-"-g -O2"}
+
+prefix=/usr/local
+exec_prefix='${prefix}'
+use_indirect_deps=no
+
+while : ; do
+  case "$1" in
+    "")
+      break
+    ;;
+    --srcdir)
+      srcdir=$2
+      shift
+    ;;
+    --prefix)
+      prefix=$2
+      shift
+    ;;
+    --exec-prefix)
+      exec_prefix=$2
+      shift
+    ;;
+    --enable-indirect-deps)
+      use_indirect_deps=yes
+    ;;
+    *)
+      echo Unknown option \"$1\".
+      exit 1
+    ;;
+  esac
+  shift
+done
+
+sed < $srcdir/build.sh.in "
+s|@configure_input@|build.sh.  Generated from build.sh.in by configure.sh.|
+s|@VERSION@|$VERSION|
+s|@srcdir@|$srcdir|
+s|@CC@|$CC|
+s|@CFLAGS@|$CFLAGS|
+s|@prefix@|$prefix|
+s|@exec_prefix@|$exec_prefix|
+s|@use_indirect_deps@|$use_indirect_deps|
+" > build.sh
+chmod +x build.sh
